### DISCUSSION TITLE
how do i remove all the extra packages installed by ml4w-hyprland-setup

### DISCUSSION THREAD

**Discussion by chaitanyasoni158** [mylinuxforwork/dotfiles#298] [Q&A]
I am  trying to  to remove hyprland altogether .

**Comment by karnaKali** [comment]
Just use the welcome app, under the hamburger menu select the uninstall script.

**Reply by Lolle2000la** [reply]
That isn't there for me.

**Reply by Rem-Titor** [reply]
me2
